clip.py

```diff
@@ -173,7 +197,7 @@ def writeData(self, clip, channel, offset, data):
             #                % (offset, data.shape[0], self.length(clip)))
 
         self.data[clip.audio_file][offset:offset + data.shape[0],
-                                   channel] = data
+        channel] = data
```

Comment:
I don't think it looks super beautiful either, but from my understanding, that's pep8